src/core/api/user.go

```diff
@@ -118,12 +118,19 @@ func (ua *UserAPI) Get() {
 		u, err := dao.GetUser(userQuery)
 		if err != nil {
 			log.Errorf("Error occurred in GetUser, error: %v", err)
-			ua.CustomAbort(http.StatusInternalServerError, "Internal error.")
+			ua.RenderFormatedError(http.StatusInternalServerError, err)
+			return
 		}
 		u.Password = ""
 		if ua.userID == ua.currentUserID {
 			u.HasAdminRole = ua.SecurityCtx.IsSysAdmin()
 		}
+		o, err := ua.getOIDCUserInfo()
```

Comment:
if ua.AuthMode != common.OIDCAuth {
		o, err := ua.getOIDCUserInfo() 
                ........
	}

Reply:
it has it in the func `getOIDCUserInfo `

Reply:
fixed